packages/nns/src/canisters/ledger/ledger.request.converts.ts

```diff
@@ -30,11 +31,14 @@ export const toTransferRawRequest = ({
   fee,
   fromSubAccount,
 }: TransferRequest): TransferRawRequest => ({
-  to: to.toNumbers(),
+  to: to.toUint8Array(),
   fee: e8sToTokens(fee ?? TRANSACTION_FEE),
   amount: e8sToTokens(amount),
   // Always explicitly set the memo for compatibility with ledger wallet - hardware wallet
   memo: memo ?? BigInt(0),
   created_at_time: [],
-  from_subaccount: fromSubAccount === undefined ? [] : [fromSubAccount],
+  from_subaccount:
+    fromSubAccount === undefined
+      ? []
+      : [arrayOfNumberToUint8Array(fromSubAccount)],
```

Comment:
The subaccount type stored in NNS Dapp canister, comes from the ledger canister. Therefore, they should be the same.

I'd say something worth adding a comment as a todo to check the types. It seems as if we are converting to one thing, to then convert it back to what we initially had.

Reply:
Link to the code: https://github.com/dfinity/nns-dapp/blob/main/rs/src/accounts_store.rs#L19

Reply:
good point, maybe that is needed first, thanks!